src/libraries/shims/src/System.csproj

```diff
@@ -7,4 +7,7 @@
   <ItemGroup>
     <Compile Include="System.forwards.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="$(CoreLibProject)" />
+  </ItemGroup>
```

Comment:
These shims projects which don't compile against references already had a reference to CoreLib before you moved CoreLib out of sfx-src.proj.

Reply:
see above comment about the sfx trimming